sql-plugin/src/main/scala/com/nvidia/spark/rapids/Plugin.scala

```diff
@@ -156,9 +156,17 @@ object RapidsPluginUtils extends Logging {
       // Derive it from spark.executor.cores, since spark.executor.cores is not set on all cluster
       // managers by default, we should judge whether if it's set explicitly.
       if (conf.contains(EXECUTOR_CORES_KEY)) {
-        conf.set(numThreadsKey, Math.max(20, conf.get(EXECUTOR_CORES_KEY).toInt).toString)
+        conf.set(numThreadsKey, Math.max(RapidsConf.MULTITHREAD_READ_NUM_THREADS_DEFAULT,
+          conf.get(EXECUTOR_CORES_KEY).toInt).toString)
+        logWarning(s"$numThreadsKey is set to " + conf.get(numThreadsKey) + ".")
```

Comment:
its a bit odd to use both string interolation and the concat.  You can use string interpolation for complex things with ${conf.get(numThreadsKey)}.  But I woudl recommend just haviung the value be in a different val:

val numThreads = Math.max(RapidsConf.MULTITHREAD_READ_NUM_THREADS_DEFAULT,
          conf.get(EXECUTOR_CORES_KEY).toInt).toString
conf.set(..., numThreads)
logWarning(.... $numThreads...)